discovery-provider/src/tasks/users.py

```diff
@@ -210,6 +210,10 @@ def parse_user_event(
             user_record.name = metadata_overrides["name"]
         if metadata_overrides["location"]:
             user_record.location = metadata_overrides["location"]
+        if metadata_overrides["collectibles"] and \
```

Comment:
maybe check if `"collectibles" in metadata_overrides` first, all the above check should probably do that as well because I believe it'll error if the key doesn't exist.

Reply:
done